db/migrate/20190912113916_create_debriefs.rb

```diff
@@ -0,0 +1,15 @@
+class CreateDebriefs < ActiveRecord::Migration[5.2]
+  def change
+    create_table :debriefs do |t|
+      t.text :notes
+      t.datetime :complete_at
+      t.bigint :completed_by, foreign_key: true
+      t.bigint :milestone_completed, foreign_key: true
+      t.references :milestone, foreign_key: true
+      t.references :iteration, foreign_key: true
+      t.date :date
```

Comment:
We need to know which milestone was associated with the debrief regardless of whether or not they completed it. Otherwise we don't know which one to show when a user looks at an old debrief.

Reply:
Isn't this comment about line 10 `t.date :date`? Makes sense about a reference to the milestone, [my other comment](https://github.com/TechforgoodCAST/fusebox-insights/pull/149#discussion_r324087964) is less about that and more about how `t.bigint :milestone_completed, foreign_key: true` is being used alongside `t.references :milestone, foreign_key: true`.